### PR TITLE
final changes are done

### PR DESCRIPTION
**Pull request by Hakob9**

### CODE REVIEW

**Review by sargsyan1986 — CHANGES_REQUESTED** [review]
not the all changes are done

File: nginx_install.sh [hunk 1]
```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
-# updating, installing and starting nginx and required libraries
+# updating apt
 apt update -y
+#checking apt version
+apt -v
```
Comment (mkrtchyan-t) [inline nginx_install.sh:6]: we don't need this line

File: nginx_install.sh [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+#installing nginx
 sudo apt install nginx -y
```
Comment (mkrtchyan-t) [inline nginx_install.sh:8]: before installing check if nginx is already installed or not

File: nginx_install.sh [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
-ufw allow 'Nginx Full'
+#checking nginx version
+nginx -v
```
Comment (mkrtchyan-t) [inline nginx_install.sh:10]: we don't need this line

File: nginx_install.sh [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
+#using uncomplicated firewall utility to configure brandmauer 
+ufw allow 'Nginx HTTP'
+#checking the changes
+ufw status
```
Comment (mkrtchyan-t) [inline nginx_install.sh:14]: we don't need this line

File: nginx_install.sh [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
+#checking how webserver services work
 systemctl start nginx
```
Comment (mkrtchyan-t) [inline nginx_install.sh:16]: should be
```
systemctl restart nginx
```

**Pull request closed without merging**